### DISCUSSION TITLE
Why not support type: 'stack' in traits

### DISCUSSION THREAD

**Discussion by chaegumi** [GrapesJS/grapesjs#4822] [Ideas]
Hi.
I think the type: 'stack' in stylemanager can also be used in traits.

![image](https://user-images.githubusercontent.com/9654650/210467764-3a40c287-3a84-41aa-a851-ccdd31a23b04.png)

![image](https://user-images.githubusercontent.com/9654650/210467791-d74fbff5-13a9-40fc-8e4e-f04ed5c5b036.png)

**Comment by zeeshan-algo** [comment]
Hi @chaegumi Can you share the code  of  how we can use the type: stack in traits

**Comment by ronaldohoch** [comment]
Nice!

**Comment by chaegumi** [comment]
https://grapesjs.com/docs/modules/Traits.html#define-new-trait-type

write a new use vue component

![image](https://user-images.githubusercontent.com/9654650/230398016-665046bc-79ec-4786-bd3d-4886371c52ec.png)

**Reply by zeeshan-algo** [reply]
Hi @chaegumi can you plzz share the code/functionality because my project require the same functionality as shown above
